src/main/java/ladder/view/LadderView.java

```diff
@@ -0,0 +1,93 @@
+package ladder.view;
+
+import ladder.domain.Line;
+import ladder.domain.Linetype;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LadderView extends TextView {
+
+    private static final String ASK_PARTICIPANT_NAME = "참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)";
+    private static final String ASK_LADDER_HEIGHT = "최대 사다리 높이는 몇 개인가요?";
+    private static final String RESULT_TEXT = "실행결과";
+    private static final String BLANK_TEXT = " ";
+    private static final int MIN_BLANK_CHECK_VALUE = 2;
+    private static final int MIN_LINE_LENGTH = 3;
+
+    @Override
+    public void view(Object o) {
+        super.view(o);
+    }
+
+    private void viewInLine(String text) {
+        System.out.print(text);
+    }
+
+    public void printAskParticipantName() {
+        view(ASK_PARTICIPANT_NAME);
+    }
+
+    public void printAskLadderHeight() {
+        view(ASK_LADDER_HEIGHT);
+    }
+
+    public void printResultText() {
+        view(RESULT_TEXT);
+    }
+
+    public void printLadder(List<String> participantNames, List<Line> lines) {
+        printResultText();
+        printParticipantNames(participantNames);
+
+        lines.stream().forEach(line -> {
```

Comment:
이 부분도 
```suggestion
        lines.forEach(line -> {
```
로 변경이 가능합니다